src/components/pager/pager.component.js

```diff
@@ -158,25 +149,21 @@ const Pager = ({
   };
 
   const renderPageSizeOptions = () => {
-    const show = I18n.t("pager.show", { defaultValue: "Show" });
-
     return (
       showPageSizeSelection && (
         <StyledPagerSizeOptionsInner>
-          {showPageSizeLabelBefore && <span>{show}</span>}
+          {showPageSizeLabelBefore && <span>{l.pager.show}</span>}
           {sizeSelector()}
-          {showPageSizeLabelAfter && <span>{records(currentPageSize)}</span>}
+          {showPageSizeLabelAfter && (
+            <span>{l.pager.records(currentPageSize)}</span>
```

Comment:
This is causing `25 records` on the left
![image](https://user-images.githubusercontent.com/2328042/126626246-852033db-9936-46be-9ca8-e26b47b93a75.png)

Reply:
changed, thanks